ansible/partials/install-homebrew.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+yes | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
I assume this would always install the latest ?  I'm wondering if we should have it install some fixed version ?

Reply:
I think with brew it's probably better to stay on latest, there's no concept of LTS versions, and bugs get found and fixed quite frequently.

Reply:
Yeah I agree with @gib on this one